src/aura/STG_CMP_ProgramPlans/STG_CMP_ProgramPlans.cmp

```diff
@@ -0,0 +1,33 @@
+<aura:component extends="c:STG_CMP_Base">
+    <aura:handler event="c:STG_EVT_Edit" action="{!c.toggleIsView}"/>
+
+    <div class="slds-grid slds-wrap">
+
+        <!-- Unique Program Plan Enforcement-->
+        <div class="slds-col slds-size--1-of-2">
+            <ui:outputText value="{!$Label.c.stgNestedPlanRequirementPP}" />
+        </div>
+        <div class="slds-col slds-size--1-of-2">
+            <div class="slds-form-element">
+                <div class="slds-form-element__control">
+                    <label class="slds-checkbox">
+                        <aura:if isTrue="{!v.isView}">
+                            <ui:outputCheckbox value="{!v.hierarchySettings.Validate_Program_Plan_for_Nested_PR__c}" class="unique-program-plan" />
+                            <aura:set attribute="else">
+                                <ui:inputCheckbox aura:id="storeErrors" value="{!v.hierarchySettings.Validate_Program_Plan_for_Nested_PR__c}"
```

Comment:
@xinnanlisf just a nit-picky thing here. Why is the aura:id and class "store-errors" here?